api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/pooling/GlobalAvgPool3D.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer.pooling
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.Layer
+import org.jetbrains.kotlinx.dl.api.core.util.TF
+import org.jetbrains.kotlinx.dl.api.inference.keras.CHANNELS_LAST
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+
+/**
+ * Global Average pooling operation for 3D data.
+ * NOTE: Works with tensors which must have rank 5 (batch_size, spatial_dim1, spatial_dim2, spatial_dim3, channels).
+ * Input shape: 5D tensor with shape `(batch_size, spatial_dim1, spatial_dim2, spatial_dim3, channels)`.
+ * Output shape: 2D tensor with shape `(batch_size, channels)`.
+ * @property [name] Custom layer name.
+ * @constructor Creates [GlobalAvgPool3D] object.
+ */
+public class GlobalAvgPool3D(
+    public val dataFormat: String = CHANNELS_LAST,
```

Comment:
Please remove dataFormat constant, we have to deal with CHANNELS_LAST by default.
I suppose it will be a common approach and refactoring for channels order in the future if it will be required.